src/miral/symbols.map

```diff
@@ -1,483 +1,479 @@
-    typeinfo?for?miral::ApplicationAuthorizer;
-    typeinfo?for?miral::ApplicationCredentials;
-    typeinfo?for?miral::ApplicationInfo;
-    typeinfo?for?miral::BasicSetApplicationAuthorizer;
-    typeinfo?for?miral::CanonicalWindowManagerPolicy;
-    typeinfo?for?miral::CommandLineOption;
-    typeinfo?for?miral::CursorTheme;
-    typeinfo?for?miral::DisplayConfiguration;
-    typeinfo?for?miral::ExternalClientLauncher;
-    typeinfo?for?miral::InternalClientLauncher;
-    typeinfo?for?miral::Keymap;
-    typeinfo?for?miral::MinimalWindowManager;
-    typeinfo?for?miral::MirRunner;
-    typeinfo?for?miral::Output;
-    typeinfo?for?miral::Output::PhysicalSizeMM;
-    typeinfo?for?miral::SetCommandLineHandler;
-    typeinfo?for?miral::SetTerminator;
-    typeinfo?for?miral::SetWindowManagementPolicy;
-    typeinfo?for?miral::StartupInternalClient;
-    typeinfo?for?miral::WaylandExtensions;
-    typeinfo?for?miral::WaylandExtensions::Builder;
-    typeinfo?for?miral::WaylandExtensions::Context;
-    typeinfo?for?miral::Window;
-    typeinfo?for?miral::WindowInfo;
-    typeinfo?for?miral::WindowManagementPolicy;
-    typeinfo?for?miral::WindowManagerOption;
-    typeinfo?for?miral::WindowManagerOptions;
-    typeinfo?for?miral::WindowManagerTools;
-    typeinfo?for?miral::WindowSpecification;
-    typeinfo?for?miral::WindowSpecification::AspectRatio;
-    typeinfo?for?miral::X11Support;
-    typeinfo?for?miral::Zone;
-    vtable?for?miral::AddInitCallback;
-    vtable?for?miral::AppendEventFilter;
-    vtable?for?miral::ApplicationAuthorizer;
-    vtable?for?miral::ApplicationCredentials;
-    vtable?for?miral::ApplicationInfo;
-    vtable?for?miral::BasicSetApplicationAuthorizer;
-    vtable?for?miral::CanonicalWindowManagerPolicy;
-    vtable?for?miral::CommandLineOption;
-    vtable?for?miral::CursorTheme;
-    vtable?for?miral::DisplayConfiguration;
-    vtable?for?miral::ExternalClientLauncher;
-    vtable?for?miral::InternalClientLauncher;
-    vtable?for?miral::Keymap;
-    vtable?for?miral::MinimalWindowManager;
-    vtable?for?miral::MirRunner;
-    vtable?for?miral::Output;
-    vtable?for?miral::Output::PhysicalSizeMM;
-    vtable?for?miral::SetCommandLineHandler;
-    vtable?for?miral::SetTerminator;
-    vtable?for?miral::SetWindowManagementPolicy;
-    vtable?for?miral::StartupInternalClient;
-    vtable?for?miral::WaylandExtensions;
-    vtable?for?miral::WaylandExtensions::Builder;
-#    vtable?for?miral::WaylandExtensions::Context;
-    vtable?for?miral::Window;
-    vtable?for?miral::WindowInfo;
-    vtable?for?miral::WindowManagementPolicy;
-    vtable?for?miral::WindowManagerOption;
-    vtable?for?miral::WindowManagerOptions;
-    vtable?for?miral::WindowManagerTools;
-    vtable?for?miral::WindowSpecification;
-    vtable?for?miral::WindowSpecification::AspectRatio;
-    vtable?for?miral::X11Support;
-    vtable?for?miral::Zone;
+    "::SetApplicationAuthorizer(typename..., Args const&...)";
```

Comment:
This is wrong: `template<typename Policy> class SetApplicationAuthorizer` is in namespace `miral`, and templates don't export symbols unless instantiated.